src/sdk/model/vm/summary/vm-summary.ts

```diff
@@ -0,0 +1,92 @@
+import { VmSummaryJson } from './__json__';
+
+/**
+ * VM Summary
+ */
+export class VmSummary {
+  private _json: VmSummaryJson;
+
+  constructor(options: VmSummary);
+  constructor(optionsJson: VmSummaryJson);
+  constructor(firstParam: VmSummary | VmSummaryJson) {
+    if (firstParam instanceof VmSummary) {
+      // Copy constructor
+      this._json = (firstParam as VmSummary).json;
+    } else {
+      // Json or empty constructor
+      this._json = (firstParam || {}) as VmSummaryJson;
+    }
+  }
+
+  /**
+   * Returns reserved CPU.
+   * @returns {number}
+   */
+  get reservedCpu(): number {
```

Comment:
Same there. If this._json is empty. this method won't return a number. Same for all other methods.